### PR TITLE
move utils to code-studio

### PR DESCRIPTION
**Pull request by Bjvanminnen**

### CODE REVIEW

File: code-studio/src/js/utils.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * Wrap a function so that it will only be executed a certain amount of time
+ * after it's originally called, to avoid multiple calls in quick succession.
+ * Subsequent calls to the returned function will reset its timer.
+ * Note: Would reuse lodash's debounce method here, but we don't appear to
+ * have access to it here.
+ * @param {function} callback - to be called after the given delay
+ * @param {number} delay - How long to wait after the _last_ call before executing the callback
+ * @returns {function}
+ */
+utils.debounce = function (callback, delay) {
```
Comment (islemaster) [inline code-studio/src/js/utils.js]: The _only_ method in here is `debounce`... can we replace this with lodash debounce now?

Comment (Bjvanminnen) [inline code-studio/src/js/utils.js]: Good idea.

**Comment by Bjvanminnen** [comment]
@islemaster updated PR. At some point we could probably just expose lodash (instead of pulling part of it out and sticking it on the dashboard namespace), but I think this is probably fine for now.

**Comment by islemaster** [comment]
:+1: LGTM.  Hopefully we'll have little enough JavaScript in dashboard that we never need to expose lodash there :grin:.

**Comment by Bjvanminnen** [comment]
good point.

**Pull request merged**